content/ghent2025/guide.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "The Beginner’s Guide to CfgMgmtCamp"
+date: 2024-12-16
+draft: false
+---
+
+# The Beginner’s Guide to CfgMgmtCamp
+
+CfgMgmtCamp is one of the longest running OSS focused DevOps conferences in the
+world. Many of the luminaries in our field cut their DevOps teeth in Ghent and
+it’s still going strong. You’ll learn so much here and make so many new exciting
+friends and spark so many ideas.
+
+But it’s community volunteer organized, free, and low budget so you may not find
+all the amenities you’re expecting from the big-budget conferences. There’s no
+concierge desk, no AI driven venue map app on your phone, not even any Ice Cube
+concerts.
+
+Instead, you’ll find an information and idea rich environment and lots of
+opportunity to engage. This guide is intended to help lower some of the
+logistical friction, so that you can focus on learning.
+
+
+1. **Getting there**
+
+    The easiest way to get to Ghent is to fly to Brussels and take the train. Head
+    down to baggage claim and you’ll find vending machines where you can purchase
+    your train ticket to the **Gent-Sint-Pieters** station. From there, see #3 for
+    local transportation.
+
+    Do not assume that you can stay in Brussels and commute to Ghent. You’ll spend a
+    lot of time on the train and you’ll miss all the social events.
+
+    1. **Coming from Fosdem**
+
+        If you’re coming from Fosdem and are traveling light, then you may be interested
```

Comment:
@KrisBuytaert you ok with this?

Reply:
Maybe point to me personally but to the CfgMgmtCamp socials ..